src/containers/DownloadCsv/index.jsx

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+
+import fetchCsv from '../../data/actions/courseEnrollments';
+import DownloadCsv from '../../components/DownloadCsv';
+
+const mapStateToProps = state => ({
+  enterpriseId: state.portalConfiguration.enterpriseId,
+  csvLoading: state.courseEnrollments.csvLoading,
+  csvError: state.courseEnrollments.csvError,
+});
+
+const mapDispatchToProps = dispatch => ({
+  fetchCsvForMethod: (enterpriseId, csvFetchMethod) => {
+    dispatch(fetchCsv(enterpriseId, csvFetchMethod));
+  },
+});
+
+export default withRouter(connect(mapStateToProps, mapDispatchToProps)(DownloadCsv));
```

Comment:
Does the `DownloadCsv` component need the router props? If not, we shouldn't include `withRouter` here.